ECMA2Yaml/ECMAHelper/Models/SDP/ItemSDPModelBase.cs

```diff
@@ -87,6 +87,10 @@ public ItemSDPModelBase()
         [YamlMember(Alias = "summary")]
         public string Summary { get; set; }
 
+        [JsonProperty("type")]
+        [YamlMember(Alias = "type")]
+        public string ChildType { get; set; }
```

Comment:
this property should be in MemberSDPModel and it's name should be just `Type`

Reply:
updated.